### PR TITLE
Add paramters and small refactoring

### LINKED ISSUE #67: Move configuration values to the config files
## Describe the solution you'd like
There are currently some configuration values that a scattered all over the code, e.g. the DSD rate for head and body behavior, the pan and tilt speed in `look_at.py` and more. They should be moved to the config file instead.

### PR DESCRIPTION
**Pull request by val-ba**
## Proposed changes

- Added parameters to `head_config.yaml`
- Used `math.radians()`
- Removed `track_post.py`

## Related issues
closes #67 

## Necessary checks
- [x] Update package version
- [ ] Run `catkin build`
- [x] Write documentation
- [x] Create issues for future work
- [ ] Test on your machine
- [ ] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

File: bitbots_head_behavior/config/head_config.yaml [hunk 1]
```diff
@@ -94,6 +103,7 @@ behavior:
     # to reduce unnecessary movement due to noise in the detection of the Ball.
     ball_tracking_min_pan_delta: 1.5
     ball_tracking_min_tilt_delta: 1.5
+    ball_tracking_look_at_speed: 3.0
```
Comment (val-ba) [inline bitbots_head_behavior/config/head_config.yaml:106]: TODO: remove

**Review by jaagut — COMMENTED** [review]
Why is the param `reduce_last_scanline` for the `search_pattern_goal` not needed?

**Comment by Flova** [comment]
> Why is the param `reduce_last_scanline` for the `search_pattern_goal` not needed?

This is done to avoid collisions with the shoulders. It was added some time ago and Valerie just moved the hard-coded param to the config.

**Comment by Flova** [comment]
Oh nvm I am stupid

**Comment by Flova** [comment]
Oh I am not stupid. I just need to read properly. The `search_pattern_goal` looks near the horizon (yes the *horizon*) and therefore doesn't need a smaller pattern at the bottom to avoid the shoulders because it's above the shoulders the entire time.

**Review by Flova — APPROVED** [review]

**Pull request merged**